dev_docs/developer_guidelines.md

```diff
@@ -41,7 +41,7 @@ The `JEST_ENV` environment variable enables the following E2E test options:
 Run end to end tests interactively during development
```

Comment:
YAY! Thanks for updating the docs too! I'll add more for the new tools introduced in the story now as well 😄 Really appreciate your help, @mweiden !! 🎉